File: official/docs/java/insurances/RetrieveInsuranceList.java
import com.easypost.EasyPost;

public class RetrieveInsuranceList {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Map<String, Object> listParams = new HashMap<>();
        listParams.put("page_size", 5);

        InsuranceCollection insurances = Insurance.all(listParams);

        System.out.println(insurances);
    }
}

File: official/docs/go/insurances/retrieve_insurance_list.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	insurances, err := client.ListInsurances(
        &easypost.ListOptions{
            PageSize: 5,
        },
    )
	
	fmt.Println(insurances)
}
